README.md

```diff
@@ -30,23 +34,45 @@ There is a script under the `dev_utils` folder that cleans up the files folder a
 ```
 Note: This script should be run every time the services are re-run
 
-The next step is to encrypt the file, using either the sda-cli downloadable [here](https://github.com/NBISweden/sda-cli/releases) and using:
+Login to LS-AAI and get the token that will be used for uploading data. For example, you can login [here](https://login.gdi.nbis), get the JWToken for the user and paste it in the `tools/proxyS3` file at the `<USER-TOKEN>` and replace the `<USER-ELIXIR-ID>` in the same file. If you are using the gdi url above, the name of the user should be on the top of the page. Copy this and replace the `@` character with `_`, then paste it in the `tools/proxyS3`.
```

Comment:
```suggestion
Login to LS-AAI and get the token that will be used for uploading data. For example, you can login [here](https://login.bp.nbis.se), get the JWT for the user and paste it in the `tools/proxyS3` file at the `<USER-TOKEN>` and replace the `<USER-ELIXIR-ID>` in the same file. If you are using the URL above, the name of the user should be on the top of the page. Copy this and replace the `@` character with `_`, then paste it in the `tools/proxyS3`.
```

URL fixes, language. But are these instructions for a developer setup or some other use? I'm wondering which use case uses tokens from LS Login.

Reply:
This was just a quick fix to get a token for uploading to `s3inbox` fast. But I see the point, so now this procedure is replaced by using a mock oidc service that runs from the compose file.